lib/models/calculator.js

```diff
@@ -69,64 +69,57 @@ module.exports = class Calculator {
 
     // Grab ELM JSON from measure, use clone so that the function added from observations does not get added over and over again
     // Set all value set versions to 'undefined' so the execution engine does not grab the specified version in the ELM
-    const elm = CalculatorHelpers.setValueSetVersionsToUndefined(_.clone(measure.elm));
-
-    // Find the main library (the library that is the "measure") and
-    // grab the version to pass into the execution engine
-    let mainLibraryVersion = '';
-    let mainLibraryIndex = 0;
-    for (let index = 0; index < elm.length; index += 1) {
-      const elmLibrary = elm[index];
-      if (elmLibrary.library.identifier.id === measure.main_cql_library) {
-        mainLibraryVersion = elmLibrary.library.identifier.version;
-        mainLibraryIndex = index;
-      }
-    }
+    const allElm = CalculatorHelpers.setValueSetVersionsToUndefined(_.clone(measure.cql_libraries.map(lib => lib.elm)));
+    // Find the main library (the library that is the "measure")
+    const mainLibraryElm = allElm.find(elm => elm.library.identifier.id === measure.main_cql_library);
 
-    const observations = measure.observations;
+    const observations = measure.population_sets[0].observations;
```

Comment:
Does this mean only the first population set will have `observations`?

Reply:
all population_sets have the same observations set on them

Reply:
Related to https://github.com/projecttacoma/cqm-execution/pull/23#discussion_r247580083 above